projects/fsastorefrontlib/src/cms-components/my-account/order/order-details/order-detail-items/order-detail-items.component.html

```diff
@@ -1,7 +1,15 @@
 <ng-container *ngIf="order$ | async as order">
-  <h2 class="heading-headline py-3 mt-3 mb-5 text-center">
-    {{ 'orderDetails.orderId' | cxTranslate }} {{ order?.code }}
-  </h2>
+  <div class="position-relative text-center">
+    <a class="vertical-center-left h5" routerLink="/my-account/orders/"
+      ><span class="d-none d-md-block">{{ 'common.back' | cxTranslate }}</span>
+      <span class="h6 d-md-none fas fa-arrow-left m-0"></span
+    ></a>
+    <h2 class="heading-headline py-3 pl-4 mt-3 mb-5 text-center">
```

Comment:
pl-4 -> px-3 since we want to have even text justification when text is longer. Also no need for text-center, since it's on wrapping div.